templates/tcs.tt

```diff
@@ -0,0 +1,52 @@
+[% IF action %]
+
+    Thanks!  We've registered your intention to [% action %] your account and hassles
+    sent to [% email %] after March 9th.
+
+[% ELSE %]
+
+    <h2 class="tc-header">The bad news</h2>
+
+    <div class="tc-section">
+        <p>From the beginning of March, mySociety will <a href="https://www.mysociety.org/2015/01/28/goodbye-to-some-old-friends/"> no longer be running several old services</a>, including HassleMe.</p>
+    </div>
+
+    <h2 class="tc-header">The good news</h2>
+
+    <div class="tc-section">
+        <p>A <strike>breakaway team of rebels</strike> group of people from mySociety
+        have decided to carry on running HassleMe, to keep the great service going
+        and to develop new features.</p>
+
+        <p>The plan is that the new team will run the site with all of the most important
+    principles that you signed up for:</p>
+    </div>
+
+    <ul class="tc-promises">
+        <li><img src="/globe.png" width="64" height="64" alt="Globe"><br>HassleMe will<br>always be free</li>
+        <li><img src="/user_circle.png" width="64" height="64" alt="User Circle"><br>HassleMe will never<br>sell your email address,<br>or send you spam</li>
+        <li><img src="/heart.png" width="64" height="64" alt="Heart"><br>HassleMe will<br>be run by people<br>who love it.</li>
```

Comment:
Please drop the content of the `alt` attributes – things for decoration should have `alt=""` :)